Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workbench/ModifyProjectManager.java

```diff
@@ -153,69 +155,68 @@ public void assignEditorRole(String projectId, String editor) throws QuadrigaSto
 	 */
 	@Override
 	@Transactional
-	public void removeEditorRole(String projectId, String editor) throws QuadrigaStorageException{
-	    ProjectEditorDTOPK projectEditorKey = new ProjectEditorDTOPK(projectId, editor);
-	    ProjectEditorDTO projectEditor = projectEditorDao.getProjectEditorDTO(projectEditorKey);
+	public void removeEditorRole(String projectId, String editor) throws QuadrigaStorageException {
+		ProjectEditorDTOPK projectEditorKey = new ProjectEditorDTOPK(projectId, editor);
+		ProjectEditorDTO projectEditor = projectEditorDao.getProjectEditorDTO(projectEditorKey);
 
-        if(projectEditor != null) {
-            projectEditorDao.deleteDTO(projectEditor);
-        }
+		if (projectEditor != null) {
+			projectEditorDao.deleteDTO(projectEditor);
+		}
 	}
-	
+
 	/*
-	 * ==================================================
-	 * Private Methods
+	 * ================================================== Private Methods
 	 * ==================================================
 	 */
 	private void deleteProjectWorkspaceMapping(ProjectDTO project) {
-        List<ProjectWorkspaceDTO> projectWorkspaceList = project.getProjectWorkspaceDTOList();
-        if(projectWorkspaceList != null) {
-            for(ProjectWorkspaceDTO projectWorkspace : projectWorkspaceList) {
-                projectWorkspaceDao.deleteDTO(projectWorkspace);
-            }
-        }
-        project.setProjectWorkspaceDTOList(null);    
-    }
-	
+		List<ProjectWorkspaceDTO> projectWorkspaceList = project.getProjectWorkspaceDTOList();
+		if (projectWorkspaceList != null) {
+			for (ProjectWorkspaceDTO projectWorkspace : projectWorkspaceList) {
+				projectWorkspaceDao.deleteDTO(projectWorkspace);
+			}
+		}
+		project.setProjectWorkspaceDTOList(null);
+	}
+
 	private void deleteProjectDictionaryMapping(ProjectDTO project) {
-        List<ProjectDictionaryDTO> projectDictionaryList = project.getProjectDictionaryDTOList();
-        if(projectDictionaryList !=null) {
-            for(ProjectDictionaryDTO projectDictionary : projectDictionaryList) {
-                projectDictionaryDao.deleteDTO(projectDictionary);;
-            }
-        }
-        project.setProjectDictionaryDTOList(null);        
-    }
-	
+		List<ProjectDictionaryDTO> projectDictionaryList = project.getProjectDictionaryDTOList();
+		if (projectDictionaryList != null) {
+			for (ProjectDictionaryDTO projectDictionary : projectDictionaryList) {
+				projectDictionaryDao.deleteDTO(projectDictionary);
+				;

```

Comment:
If this is all I have to comment, then I'll merge it but please delete this line in your next commit.